dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -545,14 +546,24 @@ public ListenableFuture<List<RouteFailure>> copy(List<Route> routes, UfsReadOpti
               AuthenticatedClientUser.set(readOptions.getUser());
             }
             checkCopyPermission(route.getSrc(), route.getDst());
+            try {
+              ValidateHandler.validate(route, writeOptions, srcFs, dstFs);
+            } catch (Throwable t) {
```

Comment:
I literally don't think any exception here would indicate a skip status

Reply:
I will change it to catch only the exception which indicates the skip status.